app/controllers/SiteMap.scala

```diff
@@ -21,6 +21,7 @@ class SiteMap(
     Ok(sitemapResponse)
   }
 
+  // TODO: more cleanup
```

Comment:
I added this comment but this is actually orthogonal to all of this stuff so you can delete it